DX-540-DX-531/Cookie-Monster-Footer-Fantasia

W3stside opened the pull request:
Cookie component + saving to localForage

WIP - needs Footer links, etc

W3stside committed:
added Cookies page to router

W3stside committed:
Cookies component pt 1

W3stside committed:
Cookies page and buttons SCSS pt 1

W3stside committed:
small SCSS fix + wrapping both cookie choices for formatting

W3stside committed:
logic update with localForage and state

Velenir commented on src/components/Cookies/index.jsx:
```diff
@@ -0,0 +1,241 @@
+
+    async componentDidMount() {
+      try {
+        this.setState({ loading: true })
```
why `loading: true` here as opposed to in constructor?

W3stside replied:
:+1:

W3stside committed:
ScrollToLink component for anchor tag movement + SCSS

W3stside committed:
DX-540 footer changes (wording, logo)

W3stside committed:
added className to wrapping app div to identify better

1. css problems when Modal needed rendering above div
2. css changes

W3stside committed:
Aux component >> Fragment + small CSS change to fix flex error in Modal

W3stside committed:
moved 'Loading' prop from async Mount to Constructor

W3stside committed:
Created and added Imprint to app

W3stside committed:
<a> to <Link>

W3stside committed:
moved styles to proper place, changed cookie wording

W3stside committed:
DX-547/React-Google-Analytics (#343)

* tokenlist without duplicates

* scrollTo(0,0) in Disclaimer submit and css changes

* 1.0.1

* production build for tagged commits

* Dx 548/update geo block (#338)

* changed netBlock logic as requirements changed

* added required countries to block - alphabetised

* changed conditional render logic in src/index

1. incorporates changes required for rinkeby/main net releases

* added main app URLS for rinkeby/main + removed ALLOWED_NETWORKS global

* refactored logic to be simpler - added network arr support

* Array.find >> Array.includes simplification

* add MKR token to rinkeby list in production (#340)

* change OR logic order (#341)

* removed Ukraine (UA) from block list

* remove circle element from DAI (#342)

* added ReactGA package and necessary code

* Check for analytics enabled and enable GA Tracking

1. types change
2. file rename

* changed component name to be more clear

* white >> blue button

W3stside committed:
removed dead code
